tests/Database/QueryBuilderTest.php

```diff
@@ -210,9 +210,7 @@ protected function getConnection($connection = null)
                 'rollBack',
                 'transactionLevel',
                 'pretend',
-            ])
-            ->addMethods([
-                'getDatabaseName',
+                'getDatabaseName'
```

Comment:
getDatabaseName is part of the interface. Phpunit complained it can't create a method that's already defined. 
By moving it to the array of methods it could be instantiated.